### PR TITLE
docs: CRAN 3.11.0

### PR DESCRIPTION
**Pull request by laresbernardo**

### CODE REVIEW

**Comment by laresbernardo** [comment]
v3.11.0 landed to CRAN after matching their requirements: https://cran.r-project.org/web/packages/Robyn/index.html

**Review by gufengzhou — APPROVED** [review]
Lgtm

File: R/R/plots.R [hunk 1]
```diff
@@ -427,29 +427,27 @@ robyn_onepagers <- function(
           sign = as.factor(ifelse(.data$xDecompPerc >= 0, "Positive", "Negative"))
         )
 
-      p2 <- suppressWarnings(
-        ggplot(plotWaterfallLoop, aes(x = .data$id, fill = .data$sign)) +
-          geom_rect(aes(
-            x = .data$rn, xmin = .data$id - 0.45, xmax = .data$id + 0.45,
```
[the remark this answers is not in the document]
Comment (gufengzhou) [reply R/R/plots.R:433]: Sure please fix it. Just make sure the order of the names are right (desc by decomp percentage)

[the remark this answers is not in the document]
Comment (laresbernardo) [reply R/R/plots.R:433]: I submitted the fix to CRAN to see if we can update it as 3.11.1. We had to get rid of the `suppressWarnings()` there.
https://github.com/facebookexperimental/Robyn/pull/1011

**Pull request merged**